modules/installation-creating-azure-bootstrap.adoc

```diff
@@ -96,10 +96,12 @@ ifdef::azure[]
 $ az deployment group create -g ${RESOURCE_GROUP} \
   --template-file "<installation_directory>/04_bootstrap.json" \
   --parameters bootstrapIgnition="${BOOTSTRAP_IGNITION}" \ <1>
-  --parameters baseName="${INFRA_ID}" <2>
+  --parameters baseName="${INFRA_ID}" \ <2>
+  --parameter bootstrapVMsize <3>
 ----
 <1> The bootstrap Ignition content for the bootstrap cluster.
 <2> The base name to be used in resource names; this is usually the cluster's infrastructure ID.
+<3> Optional. Specify the size of the bootstrap VM. 
```

Comment:
I'm also not sure if these descriptions are accurate

Reply:
It is in the sense that `--parameter bootstrapVMSize="$value"` as a whole is optional. If not specified, the default value set in the template will be used. 
I think it would be worth adding to the notes that the vm type specified has to support the desired architecture.